Atomic/scan.py

```diff
@@ -1,3 +1,7 @@
+import shutil
+import tempfile
+import docker
```

Comment:
It's used on line 548. Docker `build` can return docker.errors.APIError exception.